bin/conf

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# All config is here !!!
+#
+# By Jack @ 2015-03-23
+
+# working directory
+wd=/work/capu/CAPUBBS
```

Comment:
I don't think this `wd` is correct assigned. Can you help me to confirm it works?